crates/modelardb_compression/src/models/swing.rs

```diff
@@ -169,6 +177,18 @@ impl Swing {
                             value - maximum_deviation,
                         );
                 }
+
+                // Incremental computation of Equation 6 in Section 3.2 in the Swing and Slide paper.
```

Comment:
See my comment above about computing the slope with minimum MSE once in `model(self)` except of once per data point.